.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
-  "extends": ".eslintrc-node.json",
-  "rules": {
-	"no-nested-ternary": 1
-  }
+  	"extends": ".eslintrc-node.json",
```

Comment:
This still doesn't use tabs consistently. It uses both spaces & tabs from what I see.

Can you configure your IDE to highlight tabs & spaces so that you can see where it went wrong?

Please re-check all the new configuration files again.

Reply:
Yep, after added `.editorconfig` file and showed indentation on my IDE I got my mistake...Sorry about that. 

Please re-check all files, they should be ok, now.